### ISSUE TITLE
播放器是使用的mpv吗？视频播放速度超过4.0就没有声音了

### ISSUE THREAD

**Issue by Monster-DP** [guozhigq/pilipala#191]
播放器是使用的mpv吗？视频播放速度超过4.0就没有声音了，可以修改这个限制吗

**Comment by guozhigq** [comment]
对 使用了插件 [media_kit](https://github.com/media-kit/media-kit) 应该是基于libmpv的。没有修改这个的能力，可能需要麻烦插件作者了

**Comment by Monster-DP** [comment]
哦哦 好的 我反馈给[media-kit](https://github.com/media-kit/media-kit)了

**Comment by guozhigq** [comment]
好像是可以解决的 [When the video playback speed exceeds 4.0, there will be no sound](https://github.com/mpv-player/mpv/issues/12469)

**Comment by Monster-DP** [comment]
> 好像是可以解决的 [When the video playback speed exceeds 4.0, there will be no sound](https://github.com/mpv-player/mpv/issues/12469)
嗯嗯好的
